Documentation/Contributing/development-flow.md

```diff
@@ -256,119 +242,101 @@ Common cases that may need tests:
 * an input is specified incorrectly, for each input
 * a resource the code relies on doesn't exist, for each dependency
 
-#### Running the Integration Tests
+## Integration Tests
 
 Rook's upstream continuous integration (CI) tests will run integration tests against your changes
 automatically.
 
-You do not need to run these tests locally, but you may if you like. For instructions on how to do
-so, follow the [test instructions](https://github.com/rook/rook/blob/master/tests/README.md).
+Integration tests will be run in Github actions. If an integration test fails, a tmate session will be available for troubleshooting for a short time. See the action details for an ssh connection to the Github runner.
+
+## Commit structure
 
-### Commit structure
+Rook maintainers value clear, lengthy and explanatory commit messages.
 
-Rook maintainers value clear, lengthy and explanatory commit messages. So by default each of your commits must:
+Requirements for commits:
 
-* be prefixed by the component it's affecting, if Ceph, then the title of the commit message should be `ceph: my commit title`. If not the commit-lint bot will complain.
-* contain a commit message which explains the original issue and how it was fixed if a bug.
-If a feature it is a full description of the new functionality.
-* refer to the issue it's closing, this is mandatory when fixing a bug
-* have a sign-off, this is achieved by adding `-s` when committing so in practice run `git commit -s`. If not the DCO bot will complain.
-If you forgot to add the sign-off you can also amend a previous commit with the sign-off by running `git commit --amend -s`.
-If you've pushed your changes to GitHub already you'll need to force push your branch with `git push -f`.
+* A commit prefix from the [list of known prefixes](https://github.com/rook/rook/blob/master/.commitlintrc.json)
+* At least one paragraph that explains the original issue and the changes in the commit
+* The `Signed-off-by` tag is at the end of the commit message, achieved by committing with `git commit -s`
 
-Here is an example of an acceptable commit message:
+An example acceptable commit message:
 
 ```text
 component: commit title
 
-This is the commit message, here I'm explaining, what the bug was along with its root cause.
+This is the commit message. Here I'm explaining what the bug was along with its root cause.
 Then I'm explaining how I fixed it.
 
-Closes: https://github.com/rook/rook/issues/<NUMBER>
```

Comment:
closes is right way?

Reply:
I have never used the issue number in the commit, just in the PR, so we've never enforced this in any way. I am simplifying by removing it from this doc.